2022-01-01 12:50:51,800 INFO: Initialized curve data.
2022-01-01 12:50:51,803 INFO: Setting up initial twist data...
2022-01-01 12:50:52,872 INFO: Twist setup complete.
2022-01-01 12:50:52,872 INFO: Searching for rational points on each twist...
2022-01-01 12:50:53,676 INFO: Finished searching for rational points on each twist.
2022-01-01 12:50:53,676 INFO: Testing local solvability of twists...
2022-01-01 12:50:53,687 INFO: Finished local solvability testing.
2022-01-01 12:50:53,688 INFO: Computing elliptic curve a-invariants...
2022-01-01 12:51:05,930 INFO: Finished computation of a-invariants.
2022-01-01 12:51:05,931 INFO: Computing MW group (delta = ['4', '7', '-28', '29', '-16', '4'], g = [4, 8, -28, 29, -16, 4])...
2022-01-01 13:35:34,358 ERROR: An exception occurred. Recording data and exiting.
Traceback (most recent call last):
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 506, in <module>
    rank, MW_proven, orders, gens = twist_MW_group(D)
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 327, in twist_MW_group
    A, mw, bool_rank, bool_index = magma.MordellWeilGroup(E, nvals=_sage_const_4 )
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1787, in __call__
    return M.function_call(self._name,
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1162, in function_call
    return self._do_call(fun, nvals)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1221, in _do_call
    raise TypeError("Error executing Magma code:\n%s" % out)
TypeError: Error executing Magma code:

Current total memory usage: 7394.8MB, failed memory request: 1151.3MB
System Error: User memory limit has been reached
2022-01-03 13:14:57,000 INFO: Obsolete "loc_solv_error" keys removed.
